app/controllers/admin/members/holds_controller.rb

```diff
@@ -32,6 +32,10 @@ def destroy
       def new_hold_params
         params.require(:hold).permit(:item_id)
       end
+
+      helper_method def place_in_line_for(hold)
+        Item.find(hold.item.id).holds.index(hold) + 1
```

Comment:
This will make a query for every item on hold and instantiate a `Hold` object for every hold on that object, but most people will have less than 10 items on hold at any one time.

The optimization here would be to pull in the place in line using SQL when loading the holds, but I think that can be left until later (and if this actually becomes a performance issue).

Reply:
Makes sense. Thanks for the explanation 👍🏽